### ISSUE TITLE
Error using v1.0.2: `TypeError: Ember._registerDestructor is not a function`

### ISSUE THREAD

**Issue by dgeb** [ember-polyfills/ember-destroyable-polyfill#90]
I'm seeing a failure in a [branch](https://github.com/orbitjs/todomvc-ember-orbit/pull/16) of a simple ToDo demo of a new beta of ember-orbit. This branch fails with the following error stack: 

```
vendor.js:69026 Uncaught (in promise) TypeError: Ember._registerDestructor is not a function
    at new Cache (vendor.js:69026)
    at new Store (vendor.js:70094)
    at Function.create (vendor.js:70069)
    at FactoryManager.create (vendor.js:1236)
    at instantiateFactory (vendor.js:1029)
    at lookup (vendor.js:957)
    at processInjections (vendor.js:1079)
    at buildInjections (vendor.js:1095)
    at injectionsFor (vendor.js:1110)
    at FactoryManager.create (vendor.js:1183)
```

We've made `ember-destroyable-polyfill@1.0.2` a dependency of `ember-orbit`, so it should include this commit: https://github.com/ember-polyfills/ember-destroyable-polyfill/commit/8e2657d5025abf780a86f2af2c92fce2fd273b99 which seems to explicitly configure `Ember._registerDestructor`.

**Comment by rwjblue** [comment]
@dgeb I think you are hitting the same thing that was described in https://github.com/ember-polyfills/ember-destroyable-polyfill/issues/83. Basically the older system (in `ember-destroyable-polyfill@1`) had an async gap in time between when the code was loaded and when the `Ember._registerDestructor` function was set.

tldr; That was refactored away from for v2 and I think updating should fix for you.

**Comment by rwjblue** [comment]
@dgeb I submitted https://github.com/orbitjs/ember-orbit/pull/295 which should fix.

**Comment by dgeb** [comment]
@rwjblue Thanks so much, Rob! I confirmed that v2 does indeed fix this.
